### PR TITLE
Feature/add return view 8

### LINKED ISSUE #8: [Gerät] Create 'Rückgabeprotokoll' view

### PR DESCRIPTION
**Pull request by PhilipW97**
closes #8

### CODE REVIEW

**Comment by PhilipW97** [comment]
Bildschirmfoto 2021-06-16 um 14 10 14

File: src/main/java/com/surfSegelcenter/controller/RepairController.java [hunk 1]
```diff
@@ -1,4 +1,63 @@
...
+import org.springframework.web.bind.annotation.*;
+
+
+@Controller
 public class RepairController {
+
+	@Autowired
+	private RentService rentService;
+
+  @Autowired
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/controller/RepairController.java:28]: fix format

File: src/main/java/com/surfSegelcenter/controller/RepairController.java [hunk 2]
```diff
@@ -1,4 +1,63 @@
...
+	private RepairService repairService;
+
+  @Autowired
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/controller/RepairController.java:31]: fix format

File: src/main/java/com/surfSegelcenter/model/Rent.java [hunk 3]
```diff
@@ -98,6 +97,7 @@ public Rent(Customer customer, Sportsequipment sportsequipment, LocalDateTime st
         this.rentStatus = RentStatus.OFFEN;
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/model/Rent.java:97]: add this line:

`this.rentStatusDisplayValue = RentStatus.OFFEN.getDisplayValue();`

File: src/main/java/com/surfSegelcenter/model/Rent.java [hunk 4]
```diff
@@ -87,8 +87,7 @@
     )
     private RentStatus rentStatus;
```
Comment (pascalne2505) [inline src/main/java/com/surfSegelcenter/model/Rent.java:89]: add this:    
```
@Column(
            name = "rent_status_display_value",
            nullable = false,
            columnDefinition = "VARCHAR(10)"
    )
    private String rentStatusDisplayValue;
```

File: src/main/java/com/surfSegelcenter/model/Rent.java (hunk 3 above)
Comment (pascalne2505) [reply src/main/java/com/surfSegelcenter/model/Rent.java:97]: and add this in the "setStatus" unter "this.rentStatus = rentStatus;"

```
        if(rentStatus != null)
            this.rentStatusDisplayValue = rentStatus.getDisplayValue();
```

File: src/main/resources/templates/repair.html [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        <div></div>
+      </div>
+      <div class="contentWrapper">
+        <h2 class="heading">Reparatur hinzufügen</h2>
+        <form th:object="${repair}" class="newRentForm" th:action="@{'/repair/' + ${rentId} + '?action=add'}"
+          method="POST">
+          <input name="" th:field="*{sportsequipment}" type="hidden" />
+          <div class="inputWrapper">
+            <span>Reperaturbeschreibung</span>
+            <input th:field="*{description}" id="repairDescription" name="repairDescription" type="text"
```
Comment (pascalne2505) [inline src/main/resources/templates/repair.html:33]: i think i would put a "textarea" here instead of the input, just to be able to put more text in there

**Review by pascalne2505 — COMMENTED** [review]
at the moment its possible to edit a rent thats already closed. Should it be like this?

**Review by pascalne2505 — APPROVED** [review]
Conflicts need to be resolved

**Pull request merged**